### ISSUE TITLE
database gets hosed too easily

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/google-plugin-for-eclipse#240]
```
Using latest GPE (gae 1.8.4) and the new WST support with the Servers tab.

When I used to do development running just a standard run configuration, the 
local database wouldn't get reset unless I upgraded to a new api version. That 
was ok, cause I had control over things.

Now that I'm using WST + Servers, I'm finding that even just restarting Eclipse 
or doing a Clean... from the servers tab causes my database to get reset. It 
seems like sometimes I need to do this because some of the Servers sync'ing 
stuff can get into a funky state.

This is a pain because we have a pretty complex website that has a lot of data 
within it, even in development mode. Ideally, this database would only get 
reset when I wanted it to. We've got some scripts to generate fake data for our 
application, but those can take a while to run and are pretty system resource 
intensive.

So basically, what I'm asking for is complete control over when that database 
gets reset.
```

Original issue reported on code.google.com by `latch...@gmail.com` on 17 Sep 2013 at 4:09
